hotham/src/asset_importer/mod.rs

```diff
@@ -176,6 +207,21 @@ fn load_models_from_gltf_data(import_context: &mut ImportContext) -> Result<()>
     Ok(())
 }
 
+fn get_collider_mesh_ids(nodes: gltf::iter::Nodes) -> Vec<usize> {
+    let mut mesh_ids = Vec::new();
+    for node in nodes {
+        if !node.name().unwrap_or_default().contains(COLLIDER_TAG) {
```

Comment:
`ends_with`

Reply:
Can't do `ends_with` here! We want to see if it's *either* a wall or sensor collider, `ends_with` wouldn't meet that test.

Reply:
Wouldn't it be better to test that instead then? This will let other things through as well.

Reply:
If there is a problem with the name, eg blender added a `.001` suffix, the mesh will be silently ignored here but never added as a collider. It would be good to add a test for that actually.

Reply:
I think that's totally fine behaviour for now, honestly. This is just a first pass implementation that we can come back to in #351. 😄